constants.py

```diff
@@ -45,99 +49,117 @@ class DisplayEndingType(StrEnum):
 TEST_LOGS_PATH = Path("test_logs")
+"""The path of the folder to hold the test log files in"""
 
-# The signal to stop the logging process, this will be put in the queue to stop the process
-# see stop() and _logging_loop() for more details.
 STOP_SIGNAL = "STOP"
+"""The signal to stop the logging process, this will be put in the queue to stop the process"""
 
 
 # Formula for converting number of packets to seconds and vice versa:
 # If N = total number of packets, T = total time in seconds:
 # f = EstimatedDataPacket.frequency + RawDataPacket.frequency = 500 + 1000 = 1500 Hz
 # T = N/f => T = N/1500
 
-# Don't log more than x packets for StandbyState and LandedState
 IDLE_LOG_CAPACITY = 5000  # Using the formula above, this is 3.33 seconds of data
-# Buffer size if CAPACITY is reached. Once the state changes, this buffer will be logged to make
-# sure we don't lose data
+"""The maximum number of data packets to log in the StandbyState and LandedState. This is to prevent
+log file sizes from growing too large. Some of our 2023-2024 launches were >300 mb."""
 LOG_BUFFER_SIZE = 5000
+"""Buffer size if CAPACITY is reached. Once the state changes, this buffer will be logged to make
+sure we don't lose data"""
 
 # -------------------------------------------------------
 # State Machine Configuration
 # -------------------------------------------------------
 
 # Arbitrarily set values for transition between states:
 
-# Standby to MotorBurn:
-ACCELERATION_NOISE_THRESHOLD = 0.35  # m/s^2
+# ----------------- Standby to MotorBurn ----------------
+ACCEL_DEADBAND_METERS_PER_SECOND_SQUARED = 0.35
+"""We integrate our acceleration to get velocity, but because IMU has some noise, and other things 
+like wind or being small bumps can cause this to accumulate even while the rocket is stationary, so
+we deadband the accel to prevent this."""
 
-# We will take the magnitude of acceleration for this
-TAKEOFF_HEIGHT = 10  # meters
-TAKEOFF_VELOCITY = 10  # m/s
+TAKEOFF_HEIGHT_METERS = 10
+"""The height in meters that the rocket must reach before we consider it to have taken off."""
+TAKEOFF_VELOCITY_METERS_PER_SECOND = 10
+"""The velocity in meters per second that the rocket must reach before we consider it to have taken
+off."""
 
-# MotorBurn to Coasting:
-
-# We will only say that the motor has stopped burning if the
-# current velocity <= Max velocity * (1 - MAX_VELOCITY_THRESHOLD)
+# ---------------- MotorBurn to Coasting ----------------
 MAX_VELOCITY_THRESHOLD = 0.04
-
-# Free fall to Landing:
-MAX_FREE_FALL_LENGTH = 180.0  # seconds
-
-# Consider the rocket to have landed if it is within 15 meters of the launch site height
-# and the speed is low.
-GROUND_ALTITUDE = 10.0  # meters
-LANDED_SPEED = 5.0  # m/s
-
+"""Because motors can behave unpredictably near the end of their burn, we will only say that the
+motor has stopped burning if the current velocity <= Max velocity * (1 - MAX_VELOCITY_THRESHOLD)"""
```

Comment:
this 1 minus thing is weird. Why not just change this to 0.96?

Reply:
I like saying that it is within 4 percent

Reply:
Ok, how about `MAX_VELOCITY_THRESHOLD = 1 - 0.04`? You see it's just an unnecessary computation every time

Reply:
it being an unnecessary computation is bull, that shit probably is so negligible but you're right, makes it more complex/hard to read